domain-ee/ee-ep-merge-app/src/python_src/api.py

```diff
@@ -0,0 +1,56 @@
+import logging
+import sys
+from contextlib import asynccontextmanager
+
+import uvicorn
+from fastapi import FastAPI
+
+
+@asynccontextmanager
+async def lifespan(api: FastAPI):
+    await on_start_up()
+    yield
+    await on_shut_down()
```

Comment:
What is the purpose of defining and awaiting on these, when their implementation are both just `pass`? It feels like you want to allow them to be overridden, but I don't see how you'd do it. (It is certainly possible in Python, but it'd be awkward and non-idiomatic.)